recipes/cpython/all/conanfile.py

```diff
@@ -629,12 +647,14 @@ def _fix_install_name(self):
                     self.run("install_name_tool -change {} {} {}".format(library, new, python))
 
     def package_info(self):
+        self.cpp_info.includedirs = []
```

Comment:
This needs to be done first each component instead of the global one

Reply:
Yes I saw the the linter comment and figured as much.

I will add it to my Todo list for today